### PR TITLE
adding cards of frameworks integrations web supported

### PR DESCRIPTION
**Pull request by nelsoncode019**
Hello 👋 for me it is a pleasure to contribute to this amazing GraphQL library.

And that is why I added these cards of the most popular frameworks in the home route, I hope you like it 😁

### CODE REVIEW

File: components/integrations.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <div>
+      <p className="mt-12 mb-6 font-bold text-xl text-center">
+        Integrations for all the most popular Python development frameworks
+      </p>{" "}
+      <p>
+        Discover how to integrate Strawberry GraphQL with the most used
+        frameworks in Python for web development
+      </p>
+      <div className="grid-cards my-20 grid gap-4 px-8 lg:px-0 md:grid-cols-2 lg:gap-0 lg:grid-cols-4">
+        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4 language-python">
```
Comment (patrick91) [inline components/integrations.tsx:20]:
```suggestion
        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4">
```

File: components/integrations.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="https://strawberry.rocks/docs/integrations/flask">
```
Comment (patrick91) [inline components/integrations.tsx:21]:
```suggestion
          <a href=/docs/integrations/flask">
```

File: components/integrations.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            <FlaskIcon />
+            <h1 className=" txt-hover-framework">Flask</h1>
+          </a>
+        </div>
+        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4 language-python">
```
Comment (patrick91) [inline components/integrations.tsx:26]:
```suggestion
        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4>
```

File: components/integrations.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="https://strawberry.rocks/docs/integrations/django">
```
Comment (patrick91) [inline components/integrations.tsx:27]:
```suggestion
          <a href="/docs/integrations/django">
```

File: components/integrations.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+            <DjangoIcon />
+            <h1 className="mt-3 txt-hover-framework">Django</h1>
+          </a>
+        </div>
+        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4 language-python">
```
Comment (patrick91) [inline components/integrations.tsx:32]:
```suggestion
        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4">
```

File: components/integrations.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="https://strawberry.rocks/docs/integrations/fastapi">
```
Comment (patrick91) [inline components/integrations.tsx:33]:
```suggestion
          <a href="/docs/integrations/fastapi">
```

File: components/integrations.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+            <FastApiIcon />
+            <h1 className="mt-3 txt-hover-framework">FastAPI</h1>
+          </a>
+        </div>
+        <a href="https://strawberry.rocks/docs/integrations/sanic">
```
Comment (patrick91) [inline components/integrations.tsx:38]:
```suggestion
        <a href="/docs/integrations/sanic">
```

File: components/integrations.tsx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+          <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4 language-python">
```
Comment (patrick91) [inline components/integrations.tsx:39]:
```suggestion
          <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4">
```

File: components/integrations.tsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+            <SanicIcon />
+            <h1 className="mt-3 txt-hover-framework">Sanic</h1>
+          </div>
+        </a>
+        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4 language-python">
```
Comment (patrick91) [inline components/integrations.tsx:44]:
```suggestion
        <div className="box-frameworks font-mono overflow-x-auto border-2 border-red-500 p-6 bg-white dark:text-white dark:bg-gray-800 border-4">
```

File: components/integrations.tsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+          <a href="https://strawberry.rocks/docs/integrations/aiohttp">
```
Comment (patrick91) [inline components/integrations.tsx:45]:
```suggestion
          <a href="/docs/integrations/aiohttp">
```

**Review by patrick91 — COMMENTED** [review]
Nice work, I've added some small suggestions below :)

do you think we can make the boxes look like the ones before them: 
image
?

**Comment by nelsoncode019** [comment]
> Nice work, I've added some small suggestions below :)
> 
> do you think we can make the boxes look like the ones before them:
> image
> ?

Yes, of course, to have the same rule of styles, okay, I'll fix it

**Pull request closed without merging**